pages/standards-guidelines/wcag/index.zh-hans.md

```diff
@@ -0,0 +1,166 @@
+# Do not change the dates
+# Translate the other words below, including "Date:" and "Editor:"
+# Translate the Working Group name. Leave the Working Group acronym in English.
+footer: >
+  <p><strong>编辑:</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>在教育与推广工作组（<a href="https://www.w3.org/WAI/about/groups/eowg/">EOWG</a>）和无障碍指南工作组（<a href="https://www.w3.org/WAI/about/groups/agwg/">AG WG</a>）的意见反馈下开发。</p>
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="摘要" class="full" %}
+{:/}
+
+本页面介绍了 Web 内容无障碍指南（WCAG）国际标准，包括 WCAG 2.0、WCAG 2.1 和 WCAG 2.2。WCAG 文档解释了如何使网页内容更便于残障人士访问。
+
+另一个页面[介绍了 WCAG 3](/standards-guidelines/wcag/wcag3-intro/).
+
+WCAG 并不是无障碍的介绍。介绍内容，请参阅[无障碍基础概述](/fundamentals/).
+
+资源快速链接:
+* [如何符合 WCAG 2（快速参考）](https://www.w3.org/WAI/WCAG22/quickref/)
+* [WCAG 2.2 标准](https://www.w3.org/TR/WCAG22/), [WCAG 2.2 中的新内容](/standards-guidelines/wcag/new-in-22/)
+* [WCAG 2.1 标准](https://www.w3.org/TR/WCAG21/)
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="页面目录" class="simple" %}
+{:/}
+
+{::options toc_levels="2" /}
+
+-   This text will be replaced by the TOC.
+{:toc}
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## 简介 {#intro}
+
+Web 内容无障碍指南（WCAG）2 是通过 [W3C 流程](/standards-guidelines/w3c-process/)与全球的个人和组织合作开发的，其目标是提供一个统一的网页内容无障碍共享标准，以满足国际上个人、组织以及政府的需求。
+
+WCAG 文档阐释了如何使网页内容更便于残障人士访问。网页“内容”通常指网页或网络应用中的信息，包括：
+
+-   自然信息，例如文本，图像和声音。
+-   定义结构、呈现等的代码或标记。
+
+## WCAG 的适用对象 {#for}
+
+WCAG 面向那些需要技术标准的人群。**它并非无障碍知识的入门介绍。如需获取入门资料的链接，请查看 [常见问题中的“我该从何处入手？”](/standards-guidelines/wcag/faq/#start)。**
+
+WCAG 主要面向以下人群:
+
+-   网页内容开发者（页面作者、网站设计师等等）
+-   网页创作工具开发者
+-   网页无障碍评估工具开发者
+-   其他希望或需要制定网络无障碍标准的人，包括移动端无障碍标准的制定者
+
+为了满足其他人（包括政策制定者、管理人员和研究人员）的需求，有许多不同的 [[WAI 资源]](/resources/)。
+
+## WCAG 2 中包含了什么内容 {#whatis2}
+
+WCAG 2.2 包含 13 条指南。这些指南被归类为 [4 项原则：可感知性、可操作性、可理解性和鲁棒性](https://www.w3.org/WAI/WCAG22/Understanding/intro#understanding-the-four-principles-of-accessibility)。
+
+每条指南都有可测试的 *成功标准*。成功标准分为[三个级别：A、AA 和 AAA](https://www.w3.org/WAI/WCAG22/Understanding/conformance#levels)。
+
+成功标准是判定是否“符合”WCAG 的依据。也就是说，要符合 WCAG，内容必须满足成功标准。详细内容见 [WCAG 的符合性部分](https://www.w3.org/TR/WCAG22/#conformance)。
+
+有关 WCAG 2 指南的简要概述，可查看 **[[WCAG 2 概览]](/standards-guidelines/wcag/glance/)**。
+
+### 支持材料和补充指南 {#supplement}
+
+以下资源可以帮助您理解和实施 WCAG，并在 WCAG 的基础上进一步提升无障碍性：
+* 快速参考 / 如何符合 WCAG 2 / 检查清单
+* 理解 WCAG 2
+* WCAG 2 技术
+* WCAG 2 测试规则
+* 补充指南
+
+**请从 [WCAG 2 文档](/standards-guidelines/wcag/docs/) 中了解这些 WCAG 2 资源。**
+
+## WCAG 2.0, 2.1, 2.2 {#versions}
+
+网页内容无障碍指南（WCAG）标准在作为“W3C 推荐”网络标准发布时可供参考。 
+
+* [WCAG 2.0](https://www.w3.org/TR/WCAG20/) 于 2008 年 12 月 11 日发布。
+* [WCAG 2.1](https://www.w3.org/TR/WCAG21/) 于 2018 年 6 月 5 日发布，更新于 2023 年 9 月 21 日和 2024 年 12 月 12 日。
+* [WCAG 2.2](https://www.w3.org/TR/WCAG22/) 于 2023 年 10 月 5 日发布，并于 2024 年 12 月 12 日进行了更新。
+
+有关更新的信息，请查看 [WCAG 2 常见问题解答](/standards-guidelines/wcag/faq/)。
```

Comment:
@codersjj My automatic translation tool translates this into "For updated information, see..." while the original states "For information on the updates". The meaning is different. 

We want to indicate that information _about_ the recent updates is available in the FAQ.

Is my translating tool misleading me, or is there room from improvement of the translation?